themes/root/templates/searchapi/openapi.phtml

```diff
@@ -343,6 +352,19 @@
                     }
                 }
             },
+            "ResumptionToken": {
+                "type": "object",
+                "properties": {
+                    "token": {
+                        "description": "Token to use for the next request",
+                        "type": "string"
+                    },
+                    "expires": {
+                        "description": "Timestamp when the token expires",
```

Comment:
If this is a Unix timestamp, should the type be integer? If it's really a string, should the description clarify what date format is used?

Reply:
I believe this should be:

```
                    "expires": {
                        "description": "Timestamp when the token expires",
                        "type": "string",
                        "format": "date-time"
                    }
```
The `date-time` format is standard in [JSON Schema](https://json-schema.org/understanding-json-schema/reference/type).

Reply:
Think that could be better. Will change.

Reply:
Learning new things, managed to get whole page validated instead of part by part.

Reply:
well, had to change the * to html ent as the swagger removes * somehow